tenzir/integration/tests.yaml

```diff
@@ -263,32 +265,36 @@ tests:
         input: data/zeek/conn.log.gz
       - command: 'export ascii "where #schema == \"zeek.conn\""'
 
-  Node json zeek conn:
-    tags: [node, import-export, zeek, json]
-    steps:
-      - command: -N import -s @./misc/schema/zeek-conn.schema -t zeek.conn.custom json
-        input: data/json/conn.log.json.gz
-      - command: -N export ascii 'where duration > 6s'
-      - command: -N export ascii 'where :timestamp >= 2011-08-15T03:48'
-      - command: -N export ascii 'where :time >= 2011-08-15T03:48'
+  # This test was broken by https://github.com/tenzir/tenzir/pull/3782.
```

Comment:
Just delete it alongside all the reference files. Doesn't make sense to keep it around.